packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -576,4 +576,9 @@ instance.userData.addExecuteFileCommand({
   arguments: '--verbose -y'
 });
 asset.grantRead( instance.role );
-``` 
\ No newline at end of file
+```
+
+## Importing existing subnet
+Import existing subnet using `Subnet.fromSubId()` with limitation of not selecting subnets onePerAz.
+Making AZ property optional for `SubnetAttributes`.
```

Comment:
This not the commit message. Don't describe the act of making the change, describe the final situation for a user coming to the library for the first time.

Use longer sentences and a code sample wouldn't be amiss.

Reply:
This has not been addressed.

Reply:
updated readme